packages/unmock-core/src/service/util.ts

```diff
@@ -1,6 +1,51 @@
+import fs from "fs";
+import pointer from "json-pointer";
+import fspath from "path";
 import XRegExp from "xregexp";
 import { OAS_PATH_PARAM_REGEXP, OAS_PATH_PARAMS_KW } from "./constants";
-import { Parameter, Paths } from "./interfaces";
+import { OpenAPIObject, Parameter, Paths } from "./interfaces";
+
+/**
+ * Simple dereferencing JSON schema $ref with JSON pointers or URIs.
+ * See https://json-schema.org/understanding-json-schema/structuring.html for more.
+ */
+export function derefIfNeeded(
```

Comment:
To me it might be clearer if this was curried, with type `derefIfNeeded = ({ schema, absPath }) => (reference: any) => any` but it's a matter of taste.

Reply:
True! Forgot to update that, thanks!